src/main/java/uk/gov/pay/api/validation/ExactLengthValidator.java

```diff
@@ -1,10 +1,19 @@
 package uk.gov.pay.api.validation;
 
 
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
 import static org.apache.commons.lang3.StringUtils.isBlank;
 
-class ExactLengthValidator {
+class ExactLengthValidator implements ConstraintValidator<ValidCountry, String>  {
+    
     static boolean isValid(String value, int length) {
         return isBlank(value) || value.length() == length;
     }
+
+    @Override
+    public boolean isValid(String value, ConstraintValidatorContext context) {
+        return isValid(value, 2);
```

Comment:
It strikes me as odd to default to a length of 2 for this seemingly generic validator.
Also I'm not really sure this or the `ValidCountry` validators are necessary. Why not just add the `@Size(min = 2, max = 2, message = "Must be exactly 2 characters length")` annotation on the country field?

Reply:
Oh Dan beat me to it

Reply:
We can't use `@Size(min = 2, max = 2, message = "Must be exactly 2 characters length")` because an empty country needs to be valid as well :(
I checked with Sandor who implemented this originally and it's a requirement by Katie.